python/tvm/relay/frontend/pytorch.py

```diff
@@ -1855,6 +1855,34 @@ def one_hot(self, inputs, input_types):
     def index(self, inputs, input_types):
         data = inputs[0]
         indices = inputs[1]
+        indices_infered = self.infer_type(indices[0])
```

Comment:
I suggest enhancing relay and topi ```adv_index``` to support this, to be consistent with numpy style advanced indexing. It can be useful for other frontend as well.